debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java

```diff
@@ -280,6 +282,7 @@ private String resolveLogMineryContentQueryFromTemplate(OracleConnectorConfig co
         query = query.replace("${operationCodes}", config.isLobEnabled() ? OPERATION_CODES_LOB_ENABLED : OPERATION_CODES_LOB_DISABLED);
         query = query.replace("${schemaPredicate}", schemaReplacement == null ? "" : schemaReplacement);
         query = query.replace("${tablePredicate}", tableReplacement == null ? "" : tableReplacement);
+        System.out.println(query);
```

Comment:
This should not be here.

Reply:
Don't we have a check for forbidden APIs in place yet? If not, I'd suggest we add https://github.com/policeman-tools/forbidden-apis for 2.0.

Reply:
@gunnarmorling The checkstyle catches this, but it is error for main code only, warning for tests.

Reply:
Grrr, doh!  Thanks for the quick spot that I missed.